pkg/audit/manager.go

```diff
@@ -761,11 +792,27 @@ func (am *Manager) addAuditResponsesToUpdateLists(
 		}
 
 		totalViolationsPerEnforcementAction[ea]++
+		if enableBatch {
```

Comment:
This goes away upon removing `IsBatchingEnabled()`. However, we should still avoid appending to the list if pubsub is disabled, in order to avoid increased RAM usage.